content_handler/runtime_holder.cc

```diff
@@ -205,6 +208,33 @@ void RuntimeHolder::OnEvent(mozart::EventPtr event,
     packet.SetPointerData(0, pointer_data);
     runtime_->DispatchPointerDataPacket(packet);
     handled = true;
+  } else if (event->key_data) {
+    const char* type = nullptr;
+    if (event->action == mozart::EventType::KEY_PRESSED)
+      type = "keydown";
+    else if (event->action == mozart::EventType::KEY_RELEASED)
+      type = "keyup";
+
+    if (type) {
+      rapidjson::Document document;
+      auto& allocator = document.GetAllocator();
+      document.AddMember("type", rapidjson::Value(type, strlen(type)),
```

Comment:
I am pretty sure you are going to trip an assertion on this first `AddMember`. You have specified that the document root is an object. `document.SetObject();` should do the trick.